wifi-subsys/components/uart/src/subsys_uart.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ *
+ *
+ * @brief       uart module
+ *
+ * @copyright   Copyright (c) 2021 Mesh for all
+ * @author      xkevin190 <kevinvelasco193@gmail.com>
+ *
+ */
+#ifndef SUBSYS_UART_H
+#define SUBSYS_UART_H
+
+#include "esp_err.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+/**
+ * @brief this struct content the key and value of the AT request
+ *
+ */
+typedef struct {
+    char value[50]; /*!< AT value */
+    char key[50];   /*!< AT key */
```

Comment:
Which's the longest key?

Reply:
Anyway we shouldn't use static length arrays

Reply:
seems me a reasonable size for the keys and value

Reply:
But which is the longest key?